build/upgrade_highlightjs.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+# Run with specific branch/tag (e.g. ./upgrade_highlightjs.sh 10.6.0) or with no arguments to use master
+
+# TODO: This list of languages should be made more dynamic based on the file extensions Preview supports
```

Comment:
This seems like a likely place for future changes to break things. Should we also add a comment to where this list is maintained in the source code so both sides can hopefully stay in sync?